endorser/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es2018",
+    "module": "commonjs",
+    "outDir": "./lib",
+    "rootDir": "./src",
+    "strict": true,
+    "esModuleInterop": true
+  },
+  "exclude": [
```

Comment:
Can remove this if renaming `rootDir`, also need to add types for Jest:

`"types": ["jest"]`